backend/src/main/webapp/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
Do we have robots internally? 😃

Reply:
Ah, I don't think so. Some of this code is autogenerated by [create-react-app](https://github.com/facebook/create-react-app)